### PR TITLE
Fix for ephemeris phase axis

### PR DESCRIPTION
**Pull request by bmorris3**
Fix for phase axis `ComponentID`s that restores ephemeris support. [🐱](https://jira.stsci.edu/browse/JDAT-3690).

### CODE REVIEW

File: lcviz/marks.py
Comment (bmorris3) [inline lcviz/marks.py:1]: These changes aren't strictly necessary, but it was bothering my linter and me that we had uninitialized attrs.

File: notebooks/LCvizExample.ipynb [hunk 1]
```diff
@@ -116,7 +116,7 @@
     "eph = lcviz.plugins['Ephemeris']\n",
     "eph.period = morris2017_period\n",
     "eph.t0 = (\n",
-    "    (morris2017_epoch - reference_time).to_value(time_coord.unit) % eph.period\n",
+    "    (morris2017_epoch - reference_time).to_value(time_coordinates.unit) % eph.period\n",
```
Comment (bmorris3) [inline notebooks/LCvizExample.ipynb:119]: Sorry I didn't catch this bug earlier.

**Comment by kecnry** [comment]
Looking back at this section of code (in `ephemeris.py`), I'm a bit confused by `add_link` vs `set_links`.  Can we safely remove the individual `add_link` call and perhaps swap out `set_links(links)` with `add_link(links)`?  I'm worried otherwise we might be clearing links we intentionally set in other places - either now or down the road.

**Comment by bmorris3** [comment]
@kecnry In [b8e0bd3](https://github.com/spacetelescope/lcviz/pull/37/commits/b8e0bd32cacbaddfc66600fa1781fda8ff2bcc68) I use only `add_link`.

File: lcviz/plugins/ephemeris/ephemeris.py [hunk 2]
```diff
@@ -214,10 +214,9 @@ def _update_all_phase_arrays(self, *args, component=None):
                 )
 
                 new_links.append(new_link)
-                dc.add_link(new_link)
 
-        if len(new_links):
-            dc.set_links(new_links)
+        for new_link in new_links:
+            dc.add_link(new_link)
```
Comment (kecnry) [inline lcviz/plugins/ephemeris/ephemeris.py:219]: Despite the name, this accepts a single link or list of links

```suggestion
        dc.add_link(new_links)
```

Comment (kecnry) [reply lcviz/plugins/ephemeris/ephemeris.py:219]: (I'm just going to commit this so I can test with it... I don't think it'll be controversial 🤞 )

**Review by kecnry — APPROVED** [review]
Seems to have done the trick - thanks!

**Pull request merged**